### ISSUE TITLE
[BUG] Mac App - Chat/Text Icons

### ISSUE THREAD

**Issue by Pooch13** [ThePandemoniumInstitute/botc-release#193]
### Description

When using the Mac version of the app, I noticed that when pressing ENTER to initiate a chat/text, a bundle of  enlarged Chat/Text bubble icons flashes on the screen.

### Steps to reproduce

1. In between games
2. Press 'ENTER' to initiate chat/Text
3. Large icons flash top left corner of screen
![VideoCapture_20240705-134116](https://github.com/ThePandemoniumInstitute/botc-release/assets/174834773/2802b80b-8661-4f24-bc6c-b1d504fb2520)

### Browser

Standalone app

### Operating System

Mac

### Date and Time

July 5th, 11am PST

**Comment by bra1n** [comment]
Hey there, sorry for not getting back to you earlier! I tested it for a bit and was unable to reproduce this issue. Which version of OSX are you using? Does the bug still happen with the latest standalone version, 3.30.0?

**Comment by Pooch13** [comment]
Hello,

Sonoma 14.5
BOTC App v3.29.0

**Comment by Pooch13** [comment]
Still happening on latest version

**Comment by Pooch13** [comment]
> Hey there, sorry for not getting back to you earlier! I tested it for a bit and was unable to reproduce this issue. Which version of OSX are you using? Does the bug still happen with the latest standalone version, 3.30.0?

Still happening on latest version
![VideoCapture_20240809-131526](https://github.com/user-attachments/assets/ed4e5af0-85eb-4462-9d64-1f020c1c937c)

**Comment by bra1n** [comment]
Thanks, we'll look into this further and see whether we can reproduce it. If you have any additional info that might help us to figure out what causes it, please let us know.

**Comment by bra1n** [comment]
@Pooch13 is this still happening for you?

**Comment by Pooch13** [comment]
Yes still happening. Also today when I joined a game, all the audio cut
out. Unrelated to the current bug we are discussing

**Comment by bra1n** [comment]
Yeah, that seems to be unrelated. Were you able to find a way to consistently cause the enlarged icons to show up? Is it only ever the icons in the top left part of the circle? A step-by-step description on how to get there, ideally starting with a blank slate (e.g. from an empty session without any seats in it) would be super helpful!

**Comment by Pooch13** [comment]
I'll try to fiddle with it more today and give you all my notes.

**Comment by bra1n** [comment]
> I'll try to fiddle with it more today and give you all my notes.

Any luck so far?

**Comment by bra1n** [comment]
Hey @Pooch13, is this still an issue or can it be closed?

**Comment by Pooch13** [comment]
Hello,

Sorry I haven't used the app in a while so I am uncertain. I can check
later today and hopefully get you some more information.

**Comment by Pooch13** [comment]
Hello,

The issue still seems to be there but not nearly as noticeable.

**Comment by Pooch13** [comment]
Also I've had audio issues where all the player audio cuts out but I can
still see that players are speaking.

**Comment by bra1n** [comment]
> The issue still seems to be there but not nearly as noticeable.

What exactly does "not nearly as noticeable" mean? Could you try taking a screenshot or recording a video, please?
As for the audio issues: that's probably unrelated and might be caused by connection problems.
